pkg/operator/operator.go

```diff
@@ -39,6 +41,9 @@ const (
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	maxRetries         = 15
 	maoOwnedAnnotation = "machine.openshift.io/owned"
+
+	releaseVersionEnvVariableName = "RELEASE_VERSION"
+	releaseVersionUnknownValue    = "unknown"
```

Comment:
Is this the same value as the raw version string when it's not initialised?

Reply:
I didn't understand how/when this is used tbh, I copied it from here: https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/43facae32a53c73ad567129150db6865673bdf1d/pkg/controllers/status.go#L224

Reply:
Ok, I think it's fine to leave as that for now, lets see if anything breaks!